packages/api/src/auth/decoders/auth0.ts

```diff
@@ -15,38 +56,24 @@ import jwksClient from 'jwks-rsa'
  * }
  * ```
  *
- * You can use `sub` as a stable reference to your user, but  if you want the email
- * addres you can set a context object[^0] in rules[^1]:
+ * You can use `sub` as a stable reference to your user,
+ * but if you want the email address you can set a context object[^0] in rules[^1].
  *
  * ^0: https://auth0.com/docs/rules/references/context-object
  * ^1: https://manage.auth0.com/#/rules/new
- *
  */
-export const verifyAuth0Token = (
+export const verifyAuth0Token = async (
```

Comment:
It looks like this doesn't have to be `async`.

```suggestion
export const verifyAuth0Token = (
```